sdk.go

```diff
@@ -68,14 +68,14 @@ type Options struct {
 // New creates a new pomerium Verifier which can be used to verify a JWT token against a
 // public JWKS endpoint(s).
 func New(o *Options) (*Verifier, error) {
```

Comment:
I think `ErrDatastoreRequired` was the only error this method could return, so I'm tempted to remove the `error` return value entirely.

Any thoughts?

Reply:
yes we could remove that

Reply:
My mistake, it looks like there's also a potential error from parsing the `JWKSEndpoint` URL.